docs/source/markdown/podman-kube-down.1.md

```diff
@@ -0,0 +1,27 @@
+% podman-kube-down(1)
+
+## NAME
+podman-kube-down - Remove containers or pods based on Kubernetes YAML
+
+## SYNOPSIS
+**podman kube down** *file.yml|-*
+
+## DESCRIPTION
+**podman kube down** will read in a structured file of Kubernetes YAML. It will then tear down any pods that were based on the specifications described in the YAML and created using `podman kube play`. Any volumes that were created will remain intact. If the YAML file is specified as "-" then `podman kube down` will read the YAML from stdin.
```

Comment:
**podman kube down** reads specified Kubernetes YAML file, tearing down pods which were created by the `podman kube play` via the same Kubernetes YAML file. Any volumes that were created by the previous `podman kube play` command remain intact. If the YAML file is specified as "-"  `podman kube down` reads the YAML from stdin.